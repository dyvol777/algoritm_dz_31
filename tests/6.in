3 8
0 0 0 0 a
0 0 1 3 b
0 1 0 3 c
0 1 1 3 d
1 0 0 3 e
1 0 1 2 f
1 1 0 2 g
1 1 1 10 h