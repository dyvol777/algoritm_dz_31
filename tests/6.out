a f g 
4